src/app/public/src/modules/layout/layout-sidebar.component.html

```diff
@@ -4,6 +4,9 @@
     <stache-breadcrumbs *ngIf="showBreadcrumbs" [routes]="breadcrumbsRoutes">
     </stache-breadcrumbs>
 
+    <stache-edit-button *ngIf="showEditButton">
+    </stache-edit-button>
```

Comment:
Here's an untested idea, that might allow the user to provide their own `<stache-edit-button>` in the template:

```
<stache-edit-button *ngIf="showEditButton">
</stache-edit-button>
<ng-content
  *ngIf="!showEditButton"
  select="stache-edit-button">
 </ng-content>
```
You could simply add the above to the layouts HTML. If the user sets `showEditButton` on the wrapper, it uses the defaults, otherwise it will attempt to pull the component itself into the template.

Reply:
The trick with this one is the check for `!showEditButton`. It might be confusing if the user types `showEditButton="false"`, and then expects their custom button to show up. Maybe a check for "undefined" would be better...